gpMgmt/bin/gppylib/test/unit/data/gprebalance/unbalanced_to_spread_neg.array

```diff
@@ -0,0 +1,29 @@
+# Coordinator
+1|-1|p|p|s|u|cdw|cdw|5432|/data/coordinator
+# SDW0 (5 primaries + 3 mirrors)
+2|0|p|p|s|u|sdw0|sdw0|40000|/data/primary0
+3|1|p|p|s|u|sdw0|sdw0|40001|/data/primary1
+4|2|p|p|s|u|sdw0|sdw0|40002|/data/primary2
+5|3|p|p|s|u|sdw0|sdw0|40003|/data/primary3
+6|4|p|p|s|u|sdw0|sdw0|40004|/data/primary4
+7|8|m|m|s|u|sdw0|sdw0|50000|/data/mirror8
+8|9|m|m|s|u|sdw0|sdw0|50001|/data/mirror9
+9|10|m|m|s|u|sdw0|sdw0|50002|/data/mirror10
+# SDW1 (4 primaries + 5 mirrors)
+10|5|p|p|s|u|sdw1|sdw1|40000|/data/primary5
+11|6|p|p|s|u|sdw1|sdw1|40001|/data/primary6
+12|7|p|p|s|u|sdw1|sdw1|40002|/data/primary7
+13|8|p|p|s|u|sdw1|sdw1|40003|/data/primary8
+14|0|m|m|s|u|sdw1|sdw1|50000|/data/mirror0
+15|1|m|m|s|u|sdw1|sdw1|50001|/data/mirror1
+16|2|m|m|s|u|sdw1|sdw1|50002|/data/mirror2
+17|3|m|m|s|u|sdw1|sdw1|50003|/data/mirror3
+18|4|m|m|s|u|sdw1|sdw1|50004|/data/mirror4
+# SDW2 (3 primaries + 4 mirrors)
+19|9|p|p|s|u|sdw2|sdw2|40000|/data/primary9
+20|10|p|p|s|u|sdw2|sdw2|40001|/data/primary10
+21|11|p|p|s|u|sdw2|sdw2|40002|/data/primary11
+22|5|m|m|s|u|sdw2|sdw2|50000|/data/mirror5
+23|6|m|m|s|u|sdw2|sdw2|50001|/data/mirror6
+24|7|m|m|s|u|sdw2|sdw2|50002|/data/mirror7
+25|11|m|m|s|u|sdw2|sdw2|50003|/data/mirror11
```

Comment:
Why does the last digit in the ports and names in this file not match, while in the other files it does?